app/controllers/requisitions_controller.rb

```diff
@@ -1,7 +1,7 @@
 class RequisitionsController < ApplicationController
   before_action :set_requisition, only: [:show, :edit, :update, :destroy]
   before_action :authenticate_user!
-  before_action :authorize_requisition, except: [:show]
+  before_action :authorize_requisition, except: [:show, :print]
```

Comment:
Why do not we add 'print' to policies so that unauthorized user will not be able to print

Reply:
We can. I figured if they can access the "Show" action, they already there. But I will add it to policy

Reply:
@dawitd I have added the policy for print action. Check it out please